pkg/api/component_report.go

```diff
@@ -1007,7 +1010,9 @@ type cellStatus struct {
 }
 
 func getNewCellStatus(testID apitype.ComponentReportTestIdentification,
-	reportStatus apitype.ComponentReportStatus, existingCellStatus *cellStatus) cellStatus {
+	reportStatus apitype.ComponentReportStatus, sampleStats apitype.ComponentTestStatus,
+	existingCellStatus *cellStatus) cellStatus {
+
 	var newCellStatus cellStatus
 	if existingCellStatus != nil {
 		if (reportStatus < apitype.NotSignificant && reportStatus < existingCellStatus.status) ||
```

Comment:
With the current logic to change reportStatus based on tiraged incidents we won't add regressed tests if the reportStatus is modified due to those triaged incidents.  This includes the move to using the triagedregression statuses.  Presume we are ok here, but mentioning.